tests/kw_env_test.sh

```diff
@@ -195,6 +198,40 @@ function test_use_target_env_invalid_env()
   assertEquals "($LINENO) Env does not exists" "$?" 22
 }
 
+function test_validate_env_before_use_invalid_case_with_config()
+{
+  local output
+
+  output=$(validate_env_before_use)
```

Comment:
You are not using the value of `output` for an assertion in this test case, and it seems that it will produce part of the text in `etc/strings/env.text`.